modules/storage-persistent-storage-pv.adoc

```diff
@@ -74,7 +74,20 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 - VMware vSphere
 // - Local
 endif::openshift-enterprise,openshift-webscale,openshift-origin[]
-endif::microshift[]
+endif::microshift,openshift-rosa[]
+
+ifdef::openshift-rosa[]
+[id="types-of-persistent-volumes_{context}"]
+== Types of PVs
```

Comment:
For the Types of PVs  we generally consider it as the volume type is Filesystem  or Block . Here it seems means the csi drivers(different csi provisioners) using the title maybe a bit mislead. We could refer to `Supported CSI drivers and features in ROSA`.
Ref: https://docs.openshift.com/container-platform/4.14/storage/container_storage_interface/persistent-storage-csi.html#csi-drivers-supported_persistent-storage-csi
cc to @xueli181114

Reply:
@Phaow I have no problem changing the title. I believe it may be more accurate to mention the CSI drivers. what about something like: 
`== Supported CSI drivers for PVs` or something to that effect >?